styles/mobile.css

```diff
@@ -18,9 +22,9 @@
         display: none;
     }
 
-    .type {
-        display: none;
-    }
+    /* .type { */
```

Comment:
I prefere old code to be deleted instead of commented out.

Reply:
Aha yeh me too, 4af9f4d.